src/qtree.js

```diff
@@ -0,0 +1,436 @@
+/**
```

Comment:
Actually, I didn't see the gulpfile change that we talked about.  Was there still an issue?